Rock.Messaging/Rock.Messaging.csproj

```diff
@@ -33,6 +33,9 @@
     <DocumentationFile>bin\Release\Rock.Messaging.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
```

Comment:
I'm not sure taking on a dependency to Newtonsoft.Json is the right thing (I wrote `NamedPipeMessageSerializer` to keep the dependency out of this library). Since we're not adding the `ISerializer` interface to RockLib, all that needed to be changed was to conditionally (`#if ROCKLIB`) remove the `ISerializer` interface from the `NamedPipeMessageSerializer` class. Thoughts?